src/client/updateClientMetaInfo.js

```diff
@@ -28,22 +28,22 @@ export default function updateClientMetaInfo(options = {}, newInfo) {
     const addedTags = {}
     const removedTags = {}
 
-    Object.keys(newInfo).forEach((type) => {
+    for (const type in newInfo) {
       // ignore these
       if (metaInfoOptionKeys.includes(type)) {
-        return
+        continue
```

Comment:
What is your preference with regards to the use of `continue`? Makes it the code better readable or should we just use if..elses?

Reply:
I'm a huge fan of early returns/continue ☺️

Reply:
Yeah me too, although I find the term `continue` sometimes confusing because you could interpret it as continue the code execution (like when debugging) instead of continue with the next item